shared/src/main/scala/Elements.scala

```diff
@@ -623,6 +623,24 @@ object Elements:
         MiscHelpers.reduce(a, b)
     }
 
+    val sort: Monad = addElem(
+      Monad,
+      "S",
+      "Sort ascending",
+      List(
+        "sort",
+        "sortasc",
+        "sort-asc",
+      ),
+      "a: any -> convert to list and sort ascending",
+    ) {
+      // should do something else for num overload later
```

Comment:
I think sort returning a sorted list of digits is fine

Reply:
That's useless though

Reply:
Well I don't see what else something called "sort" could do to a number that still makes some idiomatic sense.

Reply:
That's not very useful either.